src/MockGenerateHelper.ts

```diff
@@ -0,0 +1,210 @@
+    getArrayOfItemsMock({ propertyName, items, DTOs, parseRefType }: GetArrayOfItemsMockProps): MockArrayProps {
+        let result = {
+            propertyName: `TODO: FIX ERROR in ${propertyName}`,
+            value: 'NULL',
+        } as MockArrayProps;
+
+        if (items[SwaggerProps.$ref]) {
+            const refType = items[SwaggerProps.$ref].split('/');
+
+            const ref = parseRefType(refType);
+
+            const schema = DTOs[ref];
+            if (schema && schema.enum) {
+                result = { propertyName, value: `['${schema.enum[0]}']` };
+            } else {
+                result = MockGenerateHelper.convertRefType({ propertyName, ref, isArray: true });
+            }
+        } else {
+            const type = items.oneOf ? parseRefType(items.oneOf[0][SwaggerProps.$ref].split('/')) : items.type;
+
+            if (items.oneOf) {
+                const schema = DTOs[type];
+                if (schema && schema.enum) {
+                    result = { propertyName, value: `['${schema.enum[0]}']` };
+                }
+            } else {
+                if (items.type === DataTypes.Number) {
+                    result = { propertyName, value: `[${casual.double()},${casual.double()}]` };
+                } else {
+                    result = { propertyName, value: `['${casual.word}']` };
+                }
+            }
+        }
+
+        return result;
+    }
+
+    /**
+     * Return one element of Enum or DTO
+     * @param propertyName
+     * @param oneOf
+     * @param DTOs
+     * @param parseRefType
+     */
+    getDtoMock({ propertyName, oneOf, DTOs, parseRefType }: GetArrayOfOneOfMockProps): MockArrayProps {
+        const refType = oneOf[0][SwaggerProps.$ref].split('/');
+
+        const ref = parseRefType(refType);
+
+        const schema = DTOs[ref];
+        if (schema && schema.enum) {
+            return { propertyName, value: `'${schema.enum[0]}'` };
+        } else {
+            return MockGenerateHelper.convertRefType({ propertyName, ref });
+        }
+    }
+
+    getRefTypeMock = ({ $ref, propertyName, DTOs, parseRefType }: GetRefTypeMockProps): MockArrayProps => {
+        let result = {
+            propertyName: `TODO: FIX ERROR in ${propertyName} ref:${$ref}`,
+            value: 'NULL',
+        } as MockArrayProps;
+
+        const refType = $ref.split('/');
+
+        const ref = parseRefType(refType);
+
+        const schema = DTOs[ref];
+        if (schema && schema.enum) {
+            result = { propertyName, value: `'${schema.enum[0]}'` };
+        } else if (schema) {
+            result = MockGenerateHelper.convertRefType({ propertyName, ref });
+        }
+
+        return result;
+    };
+
+    static getIsAnOrA = (word: string) => {
+        const symbol = word[0].toLowerCase();
+        const isAn =
+            symbol === 'a' || symbol === 'e' || symbol === 'i' || symbol === 'o' || symbol === 'y' || symbol === 'u';
+
+        return isAn ? 'an' : 'a';
+    };
```

Comment:
In `graphql-codegen-typescript-mock-data` we're using [indifinite](https://www.npmjs.com/package/indefinite) package to handle that
https://github.com/ardeois/graphql-codegen-typescript-mock-data/blob/master/src/index.ts#L30

Don't change it in this PR but that could be a nice improvement. However it creates breaking changes